portafly/src/components/pages/accounts/listing/AccountsListingTable.tsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react'
+import {
+  Table,
+  TableHeader,
+  TableBody,
+  OnSelect,
+  OnSort
+} from '@patternfly/react-table'
+import {
+  PageEmptyState,
+  TableEmptyState,
+  useDataListFilters,
+  useDataListPagination,
+  useDataListTable,
+  useDataListBulkActions,
+  Toolbar,
+  SearchWidget,
+  PaginationWidget,
+  BulkSelectorWidget,
+  SendEmailModal,
+  ChangeStateModal,
+  BulkActionsWidget,
+  filterRows
+} from 'components'
+import { useTranslation } from 'i18n/useTranslation'
+import { Button, ToolbarContent, ToolbarItem } from '@patternfly/react-core'
+
+const AccountsListingTable: React.FunctionComponent = () => {
+  const { t } = useTranslation('audienceAccountsListing')
+  const {
+    columns,
+    rows,
+    sortBy,
+    setSortBy,
+    selectedRows,
+    selectOne
+  } = useDataListTable()
+
+  if (rows.length === 0) {
+    return <PageEmptyState msg={t('accounts_table.empty_state')} />
+  }
+
+  const { startIdx, endIdx } = useDataListPagination()
+  const { filters, setFilters } = useDataListFilters()
+  const { modal } = useDataListBulkActions()
+
+  const options = [
+    { name: 'approved', humanName: t('actions_filter_options.by_state_options.approved') },
+    { name: 'pending', humanName: t('actions_filter_options.by_state_options.pending') },
+    { name: 'rejected', humanName: t('actions_filter_options.by_state_options.rejected') },
+    { name: 'suspended', humanName: t('actions_filter_options.by_state_options.suspended') }
+  ]
+
+  const categories = [
+    { name: 'group', humanName: t('accounts_table.group_header') },
+    { name: 'admin', humanName: t('accounts_table.admin_header') },
+    { name: 'state', humanName: t('accounts_table.state_header'), options }
+  ]
+
+  // FIXME: this throws an error
+  // useEffect(resetPagination, [filters])
+
+  const actions = {
+    sendEmail: t('shared:bulk_actions.send_email'),
+    changeState: t('shared:bulk_actions.change_state')
+  }
+
+  // FIXME: this throws an error
+  // const filteredRows = useMemo(() => (
+  //   filterRows(rows, filters, categories, columns)
+  // ), [rows, filters, sortBy])
+
+  const filteredRows = filterRows(rows, filters, columns)
+
+  const visibleRows = filteredRows.slice(startIdx, endIdx)
+
+  const onSort: OnSort = (_event, index, direction) => {
+    setSortBy(index, direction, true)
+  }
+
+  const onSelectOne: OnSelect = (_ev, selected, _rowIndex, rowData) => {
+    selectOne(rowData.id, selected)
+  }
+
+  const pagination = <PaginationWidget itemCount={filteredRows.length} />
+
+  const searchEmptyState = [{
+    heightAuto: true,
+    cells: [{
+      props: { colSpan: 8 },
+      title: <TableEmptyState
+        // title={t('accounts_table.filter_empty_state_title')}
```

Comment:
Delete comments, please

Reply:
These are strings missing from the strings files, it's there as reminder. I'll ping the docs team about it.